### PR TITLE
fix: syntax for assigning values to undefined variables

### PR DESCRIPTION
**Pull request by psturc**
### Motivation
Retrieving a value of some variable that does not exist leads to an error.
```

TASK [provision-buildfarm-apb : Copy the jenkins template] *********************
--
  | fatal: [localhost]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: No variable found with this name: JENKINS_VOLUME_SIZE"}
```
This is a problem for our nightly tests, which are using CLI to provision services - we specify only `Required` variables via CLI.

When using `lookup('env',...`, it is no longer an issue (probably could be also resolved by fixing the syntax like this: `lookup('vars', 'ANDROID_SDK_VOLUME_SIZE', default='10Gi')`)

Reference: https://docs.ansible.com/ansible/2.5/plugins/lookup/vars.html

### CODE REVIEW

**Comment by odra** [comment]
does it still need to use `default='value'` by using env instead of vars?

**Comment by wei-lee** [comment]
@psturc Those new parameters here are mainly to allow us configure some option when provision the service from the catalog. From what I can see, when the service is provisioned from the catalog with those options, they are not provided as environment variables, but they are provided as part of the '--extra-vars' flag. If we change them to look up in the "env" instead, will we be able to override the default values from cli?

Also even if those values are not set, there should be a default value. Why the default value is not working? Is it possible that the CI server is using an old version of ansible?

**Review by wei-lee — CHANGES_REQUESTED** [review]
There is a reason to look up in `vars` so that we can override them in the ansible-playbook command using `--extra-vars` flag. Not sure if the change will continue support it.

**Comment by psturc** [comment]
@odra with using `env`, no more changes are needed, because there's a different syntax for [env](https://docs.ansible.com/ansible/2.5/plugins/lookup/env.html) and [vars](https://docs.ansible.com/ansible/2.5/plugins/lookup/vars.html)
@wei-lee I see your point, we can also keep the `vars` option - I tested both (env & vars) with svcat CLI. The main reason for this PR was that the syntax for `lookup(vars` was wrong -> the default value wasn't applied if the variable wasn't defined and it resulted in error.

File: roles/provision-buildfarm-apb/defaults/main.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 android_sdk_home: "{{ lookup('env', 'ANDROID_HOME')|default('/opt/android-sdk-linux', True) }}"
```
Comment (odra) [inline roles/provision-buildfarm-apb/defaults/main.yml]: should it be using vars as well?

**Comment by psturc** [comment]
@odra @wei-lee I've set all vars to use `lookup('vars',...` and tested it with svcat CLI.

**Review by wei-lee — APPROVED** [review]
👍

**Pull request merged**